app.js

```diff
@@ -8,9 +8,12 @@ const cors = require('cors');
 
 const indexRouter = require('./routes/index');
 const apiRouter = require('./routes/api');
+const adminApiRouter = require('./routes/adminApi');
 const authRoutes = require('./routes/auth');
+const adminAuthRoutes = require('./routes/adminAuth');
 
 const auth = require('./middlewares/auth');
+const admin_auth = require('./middlewares/adminAuth');
```

Comment:
Change your variable name to camelCase style

Reply:
Fixed.